pages/shoppingCart.js

```diff
@@ -17,35 +19,69 @@ export default function ShoppingCart({ ticket }) {
     }
   }, []);
 
+  const handleDeleteTicket = (id) => {
+    const updatedCartTickets = cartTickets.filter((_, index) => index !== id);
+    setCartTickets(updatedCartTickets);
+    localStorage.setItem("cartTicket", JSON.stringify(updatedCartTickets));
+  };
+
+  const handleBuyTickets = () => {
+    setShowPopup(true);
+    setCartTickets([]);
+    localStorage.removeItem("cartTicket");
+    setTimeout(() => {
+      setShowPopup(false);
+    }, 1800);
+  };
+
   return (
     <>
       <h2>Shopping Cart Page</h2>
       <button onClick={goBack}>back</button>
 
-      {cartTickets && cartTickets.length > 0 ? (
-        <ul>
-          {cartTickets.map((ticket, id) => (
-            <li key={id}>
-              <Image
-                src={ticket.image}
-                alt="Ticket"
-                priority="high"
-                width={"280"}
-                height={"150"}
-              />
-              <p>{ticket.rating}</p>
-              <p>{ticket.name}</p>
-              <p>{ticket.location}</p>
-              <p>{ticket.date}</p>
-              <p>
-                {ticket.price} {ticket.currency}
-              </p>
-            </li>
-          ))}
-        </ul>
+      {cartTickets.length > 0 ? (
+        <>
+          <ul>
+            {cartTickets.map((ticket, id) => (
+              <li key={id}>
```

Comment:
Your `id` here is not the id 😄 
It's actually the `index`. And the `index` is not enough for the `key` prop!
Use an actual id instead! =)